WireAPI/Sources/WireAPI/APIs/UsersAPI/UsersAPIV4.swift

```diff
@@ -0,0 +1,120 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class UsersAPIV4: UsersAPIV3 {
+
+    override var apiVersion: APIVersion {
+        .v4
+    }
+
+    override func getUser(for userID: UserID) async throws -> User {
+        let request = HTTPRequest(
+            path: "\(basePath)/users/\(userID.domain)/\(userID.uuid.transportString())",
+            method: .get
+        )
+
+        let response = try await httpClient.executeRequest(request)
+
+        return try ResponseParser()
+            .success(code: 200, type: UserResponseV4.self)
```

Comment:
Is the only change the response payload? I think it would be helpful to leave a comment stating what the change is.

Reply:
Yes, new fields have been added I'll add comment on  `UserResponseV4 `